src/components/search-query.jsx

```diff
@@ -1,83 +1,95 @@
+          <strong>Election: </strong>
+          <a href={`/election/${hit.election_location.toLowerCase()}/${hit.election_date}`} target="_blank" rel="noreferrer" >
+            {hit.election_title}
+          </a>
+        </dd>
+        { hit.committee_name ?
+          <dd>
+            <strong>Committee: </strong>
+            <a href={`/committee/${hit.committee_id}`} target="_blank" rel="noreferrer" >
+              {hit.committee_name}
+            </a>
+          </dd>
+          : ''
         }
-        <a href={`/referendum/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.slug}`} target="_blank" rel="noreferrer" >
-          {hit.title}
-        </a>
-      </div>
-      : ''
-    }
-    { hit.name ?
-      <div>
-        { hit.supporting ?
-          <strong>{hit.supporting} </strong>
+        { hit.title ?
+          <dd>
+            { hit.supporting ?
+              <strong>{hit.supporting} </strong>
+              : ''
+            }
+            { hit.measure ? <strong>{hit.measure} -  </strong>
+              : <strong>Ballot Measure: </strong>
+            }
+            <a href={`/referendum/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.slug}`} target="_blank" rel="noreferrer" >
+              {hit.title}
+            </a>
+          </dd>
           : ''
         }
-        <strong>Candidate: </strong>
-        <a href={`/candidate/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.candidate_slug}`} target="_blank" rel="noreferrer" >
-          {hit.name}
-        </a>
-      </div>
-      : ''
-    }
-    { hit.office_title ?
-      <div>
-        <strong>Office: </strong>
-        <a href={`/office/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.office_slug}`} target="_blank" rel="noreferrer" >
-          {hit.office_title}
-        </a>
-      </div> : ''
-    }
-    { hit.amount ?
-      <div>
-        <strong>Amount:</strong> ${hit.amount}
+        { hit.office_title ?
+          <dd>
+            <strong>Office: </strong>
+            <a href={`/office/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.office_slug}`} target="_blank" rel="noreferrer" >
+              {hit.office_title}
+            </a>
+          </dd> : ''
+        }
+        { hit.name ?
+          <dd>
+            { hit.supporting ?
+              <strong>{hit.supporting} </strong>
+              : ''
+            }
+            <strong>Candidate: </strong>
+            <a href={`/candidate/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.candidate_slug}`} target="_blank" rel="noreferrer" >
+              {hit.name}
+            </a>
+          </dd>
+          : ''
+        }
+        { hit.amount ?
+          <dd>
+            <strong>Amount:</strong> ${hit.amount}
+          </dd>
+          : ''}
+        <dd> <strong>Election date:</strong> {hit.election_date} </dd>
+        <dd> <strong>Location:</strong> {hit.election_location} </dd>
```

Comment:
Styling nit-pick: it'd be nice to remove the space after opening curly braces `{`, and between html elements

```
{hit.amount
```
```
<dd><strong>
```